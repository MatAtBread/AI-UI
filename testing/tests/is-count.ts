/// <reference path="../test.env.d.ts"/>

for await (const x of Test.count()) {
    console.log("Count", x)
}

export { };