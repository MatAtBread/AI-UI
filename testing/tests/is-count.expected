[
  ["\"Count\"","0"],
  ["\"Count\"","1"],
  ["\"Count\"","2"],
  ["\"Count\"","3"],
  ["\"Count\"","4"],
  ["\"Count\"","5"],
  ["\"Count\"","6"],
  ["\"Count\"","7"],
  ["\"Count\"","8"],
  ["\"Count\"","9"]
]